implot3d.h

```diff
@@ -359,6 +368,8 @@ IMPLOT3D_API void SetupAxes(const char* x_label, const char* y_label, const char
 // Sets the X/Y/Z axes range limits. If ImPlotCond_Always is used, the axes limits will be locked (shorthand for two calls to SetupAxisLimits)
 IMPLOT3D_API void SetupAxesLimits(double x_min, double x_max, double y_min, double y_max, double z_min, double z_max, ImPlot3DCond cond = ImPlot3DCond_Once);
 
+IMPLOT3D_API void SetupAxisFormat(ImAxis3D idx, ImPlot3DFormatter formatter, void* data = nullptr);
```

Comment:
Can you move this function to just below `SetupAxisLimits`? It would be nice to have all `SetupAxis*` functions before the `SetupAxes*` functions.